src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -55,6 +58,7 @@ public PersonCard(Person person, int displayedIndex) {
         person.getTags().stream()
                 .sorted(Comparator.comparing(tag -> tag.tagName))
                 .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+        taskList.setText(person.getTasks().toString());
```

Comment:
I don't think this is necessary since I'm going to implement the side by side taskList window.

Reply:
Aggreed! This is only a patch for me to see if my command works. Delete it once you implement the taskList!